### DISCUSSION TITLE
Cannot read property 'getSource' of undefined

### DISCUSSION THREAD

**Discussion by dexkode4** [webpack/webpack#16543] [Q&A]
# Bug report
can't start deploy react appliction due to the following bug

**What is the current behavior?**

ERROR in webpack/runtime/consumes
Cannot read property 'getSource' of undefined
TypeError: Cannot read property 'getSource' of undefined

**Other relevant information:**
webpack version:  "^5.75.0",
npm -> 6.14.17
node -> v14.21.1
Operating System:  mac os

**Comment by alexander-akait** [comment]
Please run `npm ls webpack-sources`
